ros.ws/src/gauge_net/gauge_net/launch/gauge_net_lite.launch.py

```
@@ -0,0 +1,227 @@
+
+
+def generate_launch_description():
+    on_exit = LaunchConfiguration('on_exit', default='shutdown')
+
+    package_name = 'gauge_net'
+
+    # Get package share directory
+    package_share_dir = get_package_share_directory(package_name)
+
+    default_model_server_url = 'http://localhost:5000'
+    default_token = 'doggodoggo'
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "default_token"

(S105)